.github/workflows/push_docker.yaml

```diff
@@ -103,7 +103,7 @@ jobs:
 
   # Action pushes new docker image if we have a PR to dev and have the label benchmark
   build-push-branches:
-    if:  ${{ github.event.label.name == 'benchmark' }}
+    if:  ${{ github.event.label.name == 'benchmark required' }}
```

Comment:
One minor thing is that we could make it "benchmark complete" with a small 'b' so it is in line with other labels. Same for the maintenance label

Reply:
Nice catch, will do 👍